src/types/ElementContent.ts

```diff
@@ -1,3 +1,3 @@
-import { ReactElement } from "react";
+import { ReactNode } from "react";
 
-export type ElementContent = string | ReactElement | ReactElement[] | JSX.Element | JSX.Element[];
+export type ElementContent = string[] | ReactNode | ReactNode[];
```

Comment:
removed string, keep string[] because it throw ts errors without it  (content can be also array of contents)

Reply:
but isn't `ReactNode[]` containing `string[]`?